README.md

```diff
@@ -17,141 +17,78 @@ Woof! is a continuation of Lee Killough's Doom source port MBF targeted at moder
- * Extended HELP screens have been fixed and HELP screens are now loaded from PWADs, if available (since Woof! 5.1.0).
- * UMAPINFO lumps for MASTERLEVELS.WAD, NERVE.WAD, E1M4B.WAD and E1M8B.WAD have been added which are meant to be autoloaded with the corresponding PWAD and change map title, level transitions, par times, music and skies accordingly (since Woof! 6.0.0).
+ * The port allows to bind each action to multiple keys and mouse or joystick buttons, and unbind them using the <kbd>Del</kbd> key in the menu.
+ * The port provides modern gamepad support allowing to bind any action to shoulder triggers or analog sticks.
+
+## Capability
+
+ * The level building code has been upgraded to allow for loading maps in "extended nodes" format. Furthermore, maps using nodes in DeePBSP and (either compressed or uncompressed) ZDBSP formats can now be loaded.
+ * Support for tall textures and sprites in DeePsea format has been added.
+ * Support for unlimited extra states, sprites, mobjtypes and sounds has been added for use in Dehacked patches (supporting the "DEHEXTRA" and "DSDHacked" specs).
+ * Support for changing in-game music using the "MUSINFO" lump has been added.
+ * Support for the "UMAPINFO" lump has been added, compliant to Rev 2.0 of the spec.
+ * Support for the "MBF21" compatibility level has been added.
+
+## Compatibility
+
+ * All IWAD files since Doom 1.2 are supported, including Doom Shareware, The Ultimate Doom, Doom 2 and Final Doom. Furthermore, the IWAD files shipped with the "Doom 3: BFG Edition" and the ones published by the Freedoom project as well as Chex Quest, HACX and REKKR are supported.
+ * UMAPINFO lumps for MASTERLEVELS.WAD, NERVE.WAD, E1M4B.WAD and E1M8B.WAD have been added which are meant to be autoloaded with the corresponding PWAD and change map title, level transitions, par times, music and skies accordingly.
+ * The concept of compatibility levels has been added, currently offering "Vanilla", "Boom", "MBF" and "MBF21" (default). The default compatibility level may be changed through the menu and overridden with the `-complevel` parameter, allowing for both numeric and named arguments. Menu items in the Setup menu that don't apply to the current compatibility level are disabled and grayed out.
+ * Playback compatility with Vanilla Doom and Boom 2.02 demos has been vastly improved.
+ * The SPECHITS, REJECT and BLOCKMAP table overflow emulations have been ported over from Chocolate Doom / PrBoom+, allowing for some more obscure Vanilla demos to keep sync.
 
 ## Known issues
 
  * Savegame compatibility across different platforms and releases has only been added in Woof! 6.0.0. This version is able to read (but not save) savegames from the previous version and even MBF.exe, but may be incompatible with intermediate Woof! releases.
```

Comment:
I think this is a little confusing. I don't know how to rephrase this, but I think we should mention that the saved games are backward compatible, so users can download the new version without worrying about the saves.